### DISCUSSION TITLE
List what's missing

### DISCUSSION THREAD

**Discussion by cwills75** [a1ex4/ownfoil#116] [Ideas]
Not sure if this has been asked, but it would be nice if a base game could list what it's missing instead of just showing a yellow box for the DLC.  I have a couple of titles that I have everything for according to tinfoil.io, yet the game shows missing DLC.  All of it is showing in Ownfoil separately, but the yellow box persists.

**Comment by a1ex4** [comment]
Yes it's already on the roadmap, actually all the information needed is available in the library, it just a matter of implementing the frontend part to display it.

**Comment by cwills75** [comment]
Awesome, thanks.

**Comment by a1ex4** [comment]
Hey so that's been implemented, currently in the `develop` image if you want to try it out.

**Reply by cwills75** [reply]
There was an update for the docker and I updated it this morning, but I don't see (or don't know how) to access the new feature.  I have Repository set to:

a1ex4/ownfoil

and  Registry URL set to:

https://registry.hub.docker.com/r/a1ex4/ownfoil/tree/develop

If I pull up the console log, it shows this warning, so I think I'm on the devlop build, but I don't see a build number/version anywhere to know if it's the latest.

![Screenshot_2](https://github.com/user-attachments/assets/f4acdb50-9eab-4df7-b37c-409e95910242)

**Comment by a1ex4** [comment]
I don't know what you are using to create the container, but you should use the `a1ex4/ownfoil:develop` image tag.

**Reply by cwills75** [reply]
I changed the the repo to that and it seems to be working now, thanks.
